lms/templates/instructor/instructor_dashboard_2/cohort-editor.underscore

```diff
@@ -42,8 +42,8 @@
                 <p><%- gettext('Note: Students can be in only one cohort. Adding students to this group overrides any previous group assignment.') %></p>
             </div>
 
-            <div class="cohort-confirmations"></div>
-            <div class="cohort-errors"></div>
+            <div class="cohort-confirmations" aria-live="polite" tabindex="-1"></div>
```

Comment:
The `aria-live` attribute allows screen readers to be aware and announce any changes that happen within [this el].